src/apply.js

```diff
@@ -1029,7 +1029,11 @@ function processStyle(glStyle, mapOrGroup, styleUrl, options) {
     const type = glLayer.type;
     if (type == 'heatmap') {
       //FIXME Unsupported layer type
-      throw new Error(`${type} layers are not supported`);
+      // eslint-disable-next-line no-console
+      console.debug(
+        `[ol-mapbox-style] layers[${i}].type "${type}" not supported`
```

Comment:
I don't think it is necessary to display `[ol-mapbox-style] ` - the console shows the origin of the message.

Reply:
I've made this change, resolving.